### PR TITLE
BYOI of fireeye ax

### PR DESCRIPTION
**Pull request by Spivaka**
fixes https://github.com/demisto/server/issues/6130

### CODE REVIEW

File: Integrations/integration-fireeye.yml [hunk 1]
```diff
@@ -0,0 +1,274 @@
+commonfields:
+  id: fireeye_byoi
+  version: -1
+name: fireeye_byoi
+releaseNotes: "FireEye modified to JavaScript"
```
Comment (aviadl) [inline Integrations/integration-fireeye.yml]: I dont think we need release notes here.
But either: do `releaseNotes: "-"` then we will not add this at all
Or remove this tag, then we will take the description.
In the description you can write that now it supports the new versions of fireeye

File: Integrations/integration-fireeye.yml [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+  required: true
+- display: Username
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: false
+- display: clientToken
+  name: Client token
+  defaultvalue: ""
+  type: 0
```
Comment (aviadl) [inline Integrations/integration-fireeye.yml]: Should token be encrypted ?

File: Integrations/integration-fireeye.yml [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+            if (contentType && contentType.indexOf('application/xml') !== -1) {
+              result = JSON.parse(x2j(respoonse.Body));
+            }
+    }
+    sendRequest(commandDictionary.logout.url, commandDictionary.logout.method, token);
+    return result;
+  type: javascript
+  commands:
+  - name: fe-report
+    arguments:
```
Comment (aviadl) [inline Integrations/integration-fireeye.yml]: No context ?

Comment (Spivaka) [inline Integrations/integration-fireeye.yml]: I didn't add anything to the context yet. I want to verify this is working and replacing the go module first of all..

**Comment by Spivaka** [comment]
@aviadl please approve

**Pull request merged**